client/src/components/Autocomplete.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
```

Comment:
It looks like this is a work in progress, taken from sample code somewhere, so I won't review it until you tell me it's ready.